CMakeLists.txt

```diff
@@ -26,6 +26,21 @@ option(USE_TSAN "Enable ThreadSanitizer checks" OFF)
 option(USE_MSAN "Enable MemorySanitizer checks" OFF)
 option(USE_VALGRIND "Enable Valgrind instrumentation" OFF)
 
+set(buildTypes Release Debug RelWithDebInfo MinSizeRel)
+
+if(NOT CMAKE_BUILD_TYPE)
+    message(STATUS "No build type selected (CMAKE_BUILD_TYPE), defaulting to Release")
+    set(CMAKE_BUILD_TYPE "Release" CACHE STRING
+            "Choose the type of build, options are: Release Debug RelWithDebInfo MinSizeRel ..." FORCE)
```

Comment:
you can print here `buildTypes` variable as well, I guess

Reply:
Done